src/main/java/fr/inria/prophet4j/feature/original/OriginalRepairGenerator.java

```diff
@@ -296,7 +296,10 @@ public Repair obtainHumanRepair() {
         switch (diffEntry.type) {
             case DeleteType: // kind
                 // GuardKind: // INSERT_GUARD_RF
-                repair.kind = RepairKind.GuardKind;
+                repair.kind = RepairKind.RemoveSTMTKind;
+                if (diffEntry.srcNode instanceof CtIf) {
```

Comment:
Hi @SophieHYe 
I wonder whether we should add an additional check in this case.
Currently, it checks if a removed element is an IF, which is fine.
However, this case would also  mark as "positive" diffs that also removes the If's blocks such as:
```
- if(a>0){
-      a = b;
- }
```

In that case, I am not sure if the feature REMOVE_GUARD_RF aims at capturing that case.
I would imagine that it focus on removes of If but its block(s) then/else is not removed
```
- if(a>0){
      a = b;
-}
```
WDYT?
If we only focus on  detecting removes of the If (and not the then block), it would be necessary  to check if the Block was removed (i.e., if there is a `not root` action Delete which node is the element referenced by the If's then) or was not removed (i.e., if there is a Move action which node is is the element referenced by the If's then).

Reply:
Hi @martinezmatias, that's good point. I was only considering two remove types either they are in if block or a normal statement. I will see if I can update it to remove_whole_block or remove_only_guard or something others. I will update this PR!

Reply:
Hi @SophieHYe 
Perfect!
> I was only considering two remove types either they are in if block or a normal statement
Okey. One possibility is to simply rename the feature you create `REMOVE_GUARD_RF` by something like `REMOVE_IF_BLOCK` with the goal of avoiding confusion with the `guard` terminology.